src/App.js

```diff
@@ -1,11 +1,11 @@
 import React, { Component } from 'react';
-import ParentComponent from './components/ParentComponent/ParentComponent';
+import FormComponent from './components/FormComponent/FormComponent';
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <ParentComponent />
+       <FormComponent />
```

Comment:
Indentation is also wrong here. I'll configure linter on this project so stuff like this is a bit easier to notice and so there's no need to stress too much about these